src/openapi/peridio-admin-openapi.yaml

```diff
@@ -3829,6 +3829,9 @@ components:
       maxLength: 128
     custom-metadata:
       type: object
+      maxLength: 1048570
```

Comment:
1048570 is how many characters would fit in 1 **_mebi_**-byte (MiB).

Should be 1000000, which is how many characters would fit in 1 **_mega_**-byte (MB).